ro_help/hub/password_validation.py

```diff
@@ -0,0 +1,19 @@
+from django.utils.translation import ugettext_lazy as _
+from django.core.exceptions import ValidationError
```

Comment:
Pretty clean!

Can you please just order the imports?

Reply:
Ofc. My bad. Regarding `.po` files i talked with @catileptic and for now maybe we can just mark strings for translation and when we're closer to the finish line we can generate (or re-generate) them for the entire project?

Reply:
Sure! Let's generate them when we're almost done.